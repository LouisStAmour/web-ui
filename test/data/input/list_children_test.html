<!doctype html>
<!--
   Copyright (c) 2012, the Dart project authors.  Please see the AUTHORS file
   for details. All rights reserved. Use of this source code is governed by a
   BSD-style license that can be found in the LICENSE file.
-->
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
  <script src="packages/web_ui/testing/testing.js"></script>
</head>
<body>
  You should see the list items "a" and "b" repeated 3 times each.
  <template iterate="item in loopItemList">
    <span>{{item}}</span>
    <template if="notCond"><span>{{item}}</span></template>
    <span template if="cond">{{item}}</span>
    <span>{{item}}</span>
  </template>
  <p>Now you should see: "abc" "def" "abc" "def".
  This tests what happens when the "iterate" does not have data binding inside.
  <template iterate="item in loopItemList">
    <span>abc</span>
    <span>def</span>
  </template>
  <script type="application/dart">
    import 'dart:html';
    import 'common.dart';
    main() {
      window.postMessage('done', '*');
    }
  </script>
</body>
</html>
