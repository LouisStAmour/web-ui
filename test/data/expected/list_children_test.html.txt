Content-Type: text/plain
<html lang="en"><head><style>template { display: none; }</style>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
  <script src="../input/packages/web_ui/testing/testing.js"></script>
</head>
<body>
  You should see the list items "a" and "b" repeated 3 times each.
  <template></template>
    <span>a</span>
    <template></template><span>a</span>
    <span style="display:none"></span>
    <span>a</span>

    <span>b</span>
    <template></template><span>b</span>
    <span style="display:none"></span>
    <span>b</span>

  <p>Now you should see: "abc" "def" "abc" "def".
  This tests what happens when the "iterate" does not have data binding inside.
  <template></template>
    <span>abc</span>
    <span>def</span>

    <span>abc</span>
    <span>def</span>




</p><script type="text/javascript" src="packages/shadow_dom/shadow_dom.debug.js"></script>
<script type="application/dart" src="list_children_test.html_bootstrap.dart"></script></body></html>
